visualization/create_charts.py

```
@@ -0,0 +1,366 @@
+
+        # Final
+        if 'final' in results:
+            stages.append('Final\n(Multi-Agent)')
+            accuracies.append(results['final']['accuracy'] * 100)
+
+        # 플롯 생성
+        plt.figure(figsize=(12, 6))
+        plt.plot(stages, accuracies, marker='o', linewidth=2, markersize=10)
+
+        # 각 포인트에 수치 표시
+        for i, (stage, acc) in enumerate(zip(stages, accuracies)):
```

Ruff (0.14.13)

Loop control variable `stage` not used within loop body

Rename unused `stage` to `_stage`

(B007)